### DISCUSSION TITLE
Qual a página de documentação de APIs do Twitter?

### DISCUSSION THREAD

**Discussion by endersonmenezes** [codaqui/institucional#86] [Q&A] [answered]
Alguém pode me ajudar a achar a página de documentação de API do Twitter por favor?

**Comment by melissapalhano** [comment, marked as answer]
Seria essa aqui: <https://developer.twitter.com/en/docs/twitter-api> ?

npm: <https://www.npmjs.com/package/twitter>.

**Reply by endersonmenezes** [reply]
Ai @codaqui/membros a @melpalhano nos ajudou a encontrar a página, sempre que forem procurar algo sobre API. Pesquisem: `<produto> api` ou `<produto> for developers` assim vão conseguir encontrar se o "produto" que estão utilizando, tem facilidade para criar integrações ou não. Muito obrigado @melpalhano!
